examples/systemcalls/systemcalls.c

```diff
@@ -16,8 +21,17 @@ bool do_system(const char *cmd)
  *   and return a boolean true if the system() call completed with success 
  *   or false() if it returned a failure
 */
+	int status = 0;
+	status = system(cmd);
+	if(status == -1 )
```

Comment:
@samu7988 think about what would happen if the cmd returned with an error code other than -1 here.